content/knowledge-base/common-Google-Play-Errors.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Common Google Play Errors
+weight: 2
+---
+
+
+## Troubleshooting Common Google Play Upload Errors
+
+Codemagic enables you to automatically publish your app to the internal, alpha, beta and production tracks on Google Play. While publishing to Google play you may encounter errors if the application does not meet the release criteria. You can see the explanations of some of the common errors:
+
+- `Invalid JSON content from response: "code": 403, "message": "The caller does not have permission", "status": "PERMISSION_DENIED"`
+   - This could be due to Invalid JSON File or permission issues with service account. Please make sure you have properly set the following:
```

Comment:
```suggestion
This could be due to an invalid JSON file or permission issues with the service account. Please make sure you have done the following:
```